packages/design-picker/src/components/design-picker-category-filter/style.scss

```diff
@@ -5,6 +5,20 @@
 $design-picker-category-filter-text-color: #3c434a;
 $design-picker-category-filter-text-color-active: var(--studio-gray-100);
 
+.design-picker {
+	&.design-picker__unified {
+		.responsive-toolbar-group__dropdown {
```

Comment:
I think we can simplify this a little bit:

```diff
diff --git a/packages/design-picker/src/components/design-picker-category-filter/style.scss b/packages/design-picker/src/components/design-picker-category-filter/style.scss
index a5f8524786..b6818a80a6 100644
--- a/packages/design-picker/src/components/design-picker-category-filter/style.scss
+++ b/packages/design-picker/src/components/design-picker-category-filter/style.scss
@@ -5,17 +5,11 @@
 $design-picker-category-filter-text-color: #3c434a;
 $design-picker-category-filter-text-color-active: var(--studio-gray-100);
 
-.design-picker {
-	&.design-picker__unified {
-		.responsive-toolbar-group__dropdown {
-			&.design-picker__category-filter {
-				width: 100%;
-
-				.responsive-toolbar-group__grouped-list {
-					justify-content: flex-start;
-				}
-			}
-		}
+.design-picker__category-filter.responsive-toolbar-group__dropdown {
+	width: 100%;
+
+	.responsive-toolbar-group__grouped-list {
+		justify-content: flex-start;
 	}
 }

```

Reply:
Yeah, it seems to work. Changed.